_posts/2019-01-15-recursion.md

```diff
@@ -0,0 +1,312 @@
+---
+author: Sophie DeBenedetto
+author_link: https://github.com/sophiedebenedetto
+categories: recursion
+date: 2019-01-15
```

Comment:
@SophieDeBenedetto can we update this to the date you want to publish?  We could shoot for tomorrow.